tool/test/release_notes/release_note_classes_test.dart

```diff
@@ -0,0 +1,15 @@
+// import 'package:devtools_repo/release_note_classes.dart';
```

Comment:
Is this file meant to be here?

Reply:
remove this file then LGTM

Reply:
this was accidentally added from another branch i was working on. Sorry for the confusion